src/screens/tenants/CreateTenantDialog.tsx

```diff
@@ -46,24 +47,27 @@ export default class CreateTenantDialog extends React.Component<Props, State> {
 
   public constructor(props: Props) {
     super(props);
-    this.tenantEndpointClouds = Configuration.ENDPOINT_CLOUDS;
+    let tenantEndpointClouds: EndpointCloud[];
+    if (__DEV__) {
+      tenantEndpointClouds = Configuration.ENDPOINT_CLOUDS_DEV;
+    } else {
+      tenantEndpointClouds = Configuration.ENDPOINT_CLOUDS;
+    }
+    const newTenantEndpointCloudId = tenantEndpointClouds.find((tenantEndpointCloud) => tenantEndpointCloud.id === 'scp').id;
```

Comment:
Maybe add a chaining operator before the .id ? I think find returns undefined if nothing is found